README.md

```diff
@@ -1,67 +1,67 @@
 # CTO.ai GitHub Action
 
-The CTO.ai GitHub Action makes it easy to integrate GitHub into the CTO.ai
-Delivery Metrics platform.
+The CTO.ai GitHub Action makes it easy to extend CTO.ai Delivery Insight 
+with specific events from your Github Action, such as deployments.
 
-## CTO.ai Drop-In GitHub Action Agent
 
-The CTO.ai Drop-In GitHub Agent will automatically record some CTO.ai Delivery
-& LifeCycle Metrics that can then be viewed on the CTO.ai Delivery Metrics
-dashboard.
-
-### Install Agent
+### Install Default Agent
 
 1. From your GitHub repo, click Settings -> Secrets -> New Secret
   - Create CTOAI_TEAM_ID secret using your CTO.ai-issued Team Id.
   - Create CTOAI_EVENTS_API_TOKEN secret using your CTO.ai-issued API Token.
-2. Access the CTO.ai Agent workflow from:
-  - https://github.com/cto-ai/agent/blob/master/.github/workflows/ctoai-agent.yml
-3. Place `ctoai-agent.yml` in the workflows folder of your repo:
-  - .github/workflows/ctoai-agent.yml
-4. You're done! Delivery & LifeCycle Metrics can now be viewed on the Metrics
-   Dashboard.
+2. Copy all of the CTO.ai Agent workflow from:
+  - https://github.com/cto-ai/agent/blob/master/.github/workflows/
+3. Place these workflows in the workflows folder of your repo:
+  - .github/workflows/ or integrate directly with your existing actions.
+4. You're done! Now events from your workflow will be sent!
+
+By default our existing agents assume you use Github Flow. By default we count 
+commits to master as a "successful deployment" and pull requests with status closed 
+as a "failed deployment". However, if you want to get more accurate with measuring your
+actual deployments you should manually instrument your pipeline at this custom stage.
```

Comment:
should avoid copy/paste, let people integrate our action into their workflows

Reply:
@mau had similar feelings about the templates, which we should think about.

Reply:
We'll need to keep a default working example that people can drop in that "just works".

Manual implementation of the action is harder to mock here but it's covered in the README.

This will definitely be something we need to further document as the question has already come up.

Reply:
I would remove the point about "closed pull_request as failed deployment"